srcpkgs/audiofile/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'audiofile'
 pkgname=audiofile
 version=0.3.6
-revision=2
-wrksrc=$pkgname-$pkgname-$version
+revision=3
+wrksrc="$pkgname-$pkgname-$version"
 build_style=gnu-configure
 hostmakedepends="automake libtool asciidoc pkg-config"
-makedepends="gtk+-devel alsa-lib-devel"
+makedepends="alsa-lib-devel libflac-devel"
 short_desc="C library for reading and writing audio files"
 maintainer="Michael Aldridge <maldridge@VoidLinux.eu>"
-license="LGPL-2.1"
+license="LGPL-2.1-or-later"
 homepage="http://audiofile.68k.org"
-disable_parallel_build=1
-distfiles="http://github.com/mpruett/audiofile/archive/audiofile-$version.tar.gz"
+distfiles="http://github.com/mpruett/audiofile/archive/audiofile-${version}.tar.gz"
 checksum=52125fee6c7454d743acdc27ebda194c6b5c7b9111426c7d5fdea0754cd366cc
+disable_parallel_build=1
 
 pre_configure() {
 	./autogen.sh
```

Comment:
https://github.com/mpruett/audiofile/blob/audiofile-0.3.6/autogen.sh#L52
`autogen.sh` tries to run `./configure` directly, this seems wrong.

Reply:
Switched to autoreconf -fi, there is no apparent difference between both.


#### rant

really autogen.sh should only do stuff that you can't do with autoreconf -fi like setting some vital  files somewhere